packages/program/src/functions/base-invocation-scope.ts

```diff
@@ -292,8 +293,19 @@ export class BaseInvocationScope<TReturn = any> {
     assert(this.program.account, 'Wallet is required!');
 
     const transactionRequest = await this.getTransactionRequest();
+    const { maxFee, gasUsed, minGas } = await this.getTransactionCost();
```

Comment:
1. Will this add an extra request to the node?
 1. Which data is asked?

Please note the problems we are addressing on this parallel quest:
 - https://github.com/FuelLabs/fuels-ts/issues/1706

We need to be very careful.

Reply:
@arboleya No, this won't add any _new_ requests. It just uses more data from the request we are already making as of now.

Reply:
@arboleya just confirmed. All good. No new calls to the client